core/editor.js

```diff
@@ -585,6 +585,11 @@
 				// language has been resolved to this plugin.
 				languageCodes.push( lang );
 
+				// Plugin is supported by default (#2692).
+				plugin.isSupportedEnvironment = plugin.isSupportedEnvironment || function() {
+					return true;
+				};
```

Comment:
`onLoad` method may not be provided, but yeah it makes more sense to attach default method implementation inside `core/plugins.js` file.

Reply:
> `onLoad` method may not be provided.

Yes, I meant the line above. It shouldn't depend on presence of `onLoad`.

Reply:
Looks like the safest place is https://github.com/ckeditor/ckeditor-dev/blob/14e3bab99f1d4b968dd44e9b4e5b408e97b26758/core/plugins.js#L37 because it makes sure that the function won't be attached multiple times.